src/api/app/views/layouts/webui/webui.html.erb

```diff
@@ -9,6 +9,13 @@
     <%= favicon_link_tag %>
 
     <title><%= "#{@pagetitle} - " if @pagetitle -%><%= home_title %></title>
+    <meta property="og:title" content="<%= meta_title %>"/>
+    <meta property="og:type" content="website"/>
+    <meta property="og:url" content="<%= request.original_url %>"/>
+    <meta property="og:image" content="<%= meta_image %>"/>
+    <% if meta_description %>
+    <meta property="og:description" content="<%= meta_description %>"/>
```

Comment:
This line should be indented with 2 spaces.

Reply:
Done!